generation_config.yaml

```diff
@@ -681,7 +681,6 @@ libraries:
   product_documentation: https://cloud.google.com/dataform/docs
   api_description: Help analytics teams manage data inside BigQuery using SQL.
   GAPICs:
-  - proto_path: google/cloud/dataform/v1alpha2
```

Comment:
Thanks @zhumin8 for creating the fix! I thought our default behavior is that if the path does not exist, we would throw errors, @JoeWang1127 can you confirm it? If this is not expected, I think we should create an issue in the backlog to track it.

Reply:
It does throw no such file error, if that's what you mean: https://github.com/googleapis/google-cloud-java/pull/11462#issuecomment-2715669430

Reply:
OK, that makes sense then. It was by design that we don't want to automatically remove a deprecated module, because it is technically a breaking change.
Can you please change the title to `feat!` to indicate that it's a breaking change?